Marlin/Marlin_main.cpp

```diff
@@ -756,6 +756,7 @@ void loop() {
           // M29 closes the file
           card.closefile();
           SERIAL_PROTOCOLLNPGM(MSG_FILE_SAVED);
+          ok_to_send();
```

Comment:
@thinkyhead 
Take an other look on when `process_next_command()` is called and when not.